app/services/idv/steps/verify_step.rb

```diff
@@ -4,7 +4,20 @@ class VerifyStep < VerifyBaseStep
       STEP_INDICATOR_STEP = :verify_info
 
       def call
-        enqueue_job
+        if current_async_state.none?
+          enqueue_job
+          render_pending_response
+        elsif current_async_state.in_progress?
+          render_pending_response
+        elsif current_async_state.missing?
+          delete_async
+          @flow.analytics.idv_proofing_resolution_result_missing
+          render_json({ error: I18n.t('idv.failure.timeout') })
+          FormResponse.new(success: false)
+        elsif current_async_state.done?
+          render_json({ redirect_url: idv_url })
```

Comment:
can drop the optional curly braces, if we want:
```suggestion
          render_json(error: I18n.t('idv.failure.timeout'))
          FormResponse.new(success: false)
        elsif current_async_state.done?
          render_json(redirect_url: idv_url)
```

Reply:
Can't actually, since the JSON argument is the first of two arguments:

https://github.com/18F/identity-idp/blob/12cb492ead38c6e50836cf21a33db22b569cc6f0/app/services/flow/base_step.rb#L73-L75

Could look into updating that to make it a bit more ergonomic.

Reply:
:oh-no: .... oof I forgot about that Ruby 3 syntax change...

Maybe clearer to have them both be kwargs: `render_json(body:, status: nil)`? But also not a huge deal

Reply:
At that point I think we may just as well use the base Rails `render json:, status:` which... maybe we should.

Then again, I'm trying to decide how best to handle the "submit" events for polling requests, and one of the ideas I had is for `render_json` to detect `:accepted` status as a pending result and not log it as a normal submission event.

Reply:
> At that point I think we may just as well use the base Rails `render json:, status:` which... maybe we should.

This isn't feasible within the FlowStateMachine, since the reason it exists is due to the fact there's a number of extra steps to get that JSON to be rendered by the FlowStateMachine apparatus.

Reply:
Yeah I had a feeling the FSM's "I'm a controller but not a controller" would get in the way 😭 

Changing this method/style is not a high priority, I'm going to resolve this comment so it's not a distraction